packages/block-library/src/spacer/edit.js

```diff
@@ -51,50 +61,122 @@ const SpacerEdit = ( {
+		onResizeStop();
+		const spacerWidth = Math.min(
+			parseInt( width + delta.width, 10 ),
+			MAX_SPACER_WIDTH
+		);
+		updateWidth( spacerWidth );
+		setIsResizing( false );
+	};
+
+	const resizableBoxWithOrientation = ( blockOrientation ) => {
+		if ( blockOrientation === 'horizontal' ) {
+			return (
 				<ResizableBox
 					className={ classnames(
 						'block-library-spacer__resize-container',
+						'resize-horizontal',
 						{
 							'is-selected': isSelected,
 						}
 					) }
 					size={ {
-						height,
+						width,
+						height: 32,
 					} }
-					minHeight={ MIN_SPACER_HEIGHT }
+					minWidth={ MIN_SPACER_WIDTH }
 					enable={ {
 						top: false,
-						right: false,
-						bottom: true,
+						right: true,
+						bottom: false,
 						left: false,
 						topRight: false,
 						bottomRight: false,
 						bottomLeft: false,
 						topLeft: false,
 					} }
 					onResizeStart={ handleOnResizeStart }
-					onResizeStop={ handleOnResizeStop }
+					onResizeStop={ handleOnHorizontalResizeStop }
 					showHandle={ isSelected }
 					__experimentalShowTooltip={ true }
 					__experimentalTooltipProps={ {
-						axis: 'y',
-						position: 'bottom',
+						axis: 'x',
+						position: 'corner',
 						isVisible: isResizing,
 					} }
 				/>
+			);
+		}
+
+		return (
+			<ResizableBox
+				className={ classnames(
+					'block-library-spacer__resize-container',
+					{
+						'is-selected': isSelected,
+					}
+				) }
+				size={ {
+					height,
+				} }
+				minHeight={ MIN_SPACER_HEIGHT }
+				enable={ {
+					top: false,
+					right: false,
+					bottom: true,
+					left: false,
+					topRight: false,
+					bottomRight: false,
+					bottomLeft: false,
+					topLeft: false,
+				} }
+				onResizeStart={ handleOnResizeStart }
+				onResizeStop={ handleOnVerticalResizeStop }
+				showHandle={ isSelected }
+				__experimentalShowTooltip={ true }
+				__experimentalTooltipProps={ {
+					axis: 'y',
+					position: 'bottom',
+					isVisible: isResizing,
+				} }
+			/>
+		);
+	};
+
+	useEffect( () => {
+		if ( orientation === 'horizontal' && ! width ) {
+			updateWidth( 72 );
+			updateHeight( 0 );
+		}
+	}, [] );
+
+	return (
+		<>
+			<View { ...useBlockProps() }>
+				{ resizableBoxWithOrientation( orientation ) }
 			</View>
 			<InspectorControls>
 				<PanelBody title={ __( 'Spacer settings' ) }>
-					<RangeControl
-						label={ __( 'Height in pixels' ) }
-						min={ MIN_SPACER_HEIGHT }
-						max={ Math.max( MAX_SPACER_HEIGHT, height ) }
-						value={ height }
-						onChange={ updateHeight }
-					/>
+					{ orientation === 'horizontal' && (
+						<RangeControl
+							label={ __( 'Width in pixels' ) }
+							min={ MIN_SPACER_WIDTH }
+							max={ Math.max( MAX_SPACER_WIDTH, width ) }
+							value={ width }
+							onChange={ updateWidth }
+						/>
+					) }
+					{ orientation !== 'horizontal' && (
+						<RangeControl
+							label={ __( 'Height in pixels' ) }
```

Comment:
General question from me, when do we normally add inline translator comments?

Reply:
Comments are used to add information about the string's placement or meaning in situations where the text content, taken out of its context, might not be clear enough. This is because translators only see a list of strings and don't necessarily know whereabouts in the UI they might live.

Reply:
Gotcha, so more of a case by case basis if it's pretty short/generic 👍